reference/docs-conceptual/install/Installing-PowerShell-Core-on-macOS.md

```diff
@@ -1,12 +1,13 @@
 ---
 title: Installing PowerShell on macOS
 description: Information about installing PowerShell on macOS
-ms.date: 07/30/2020
+ms.date: 08/24/2020
 ---
 
 # Installing PowerShell on macOS
 
-PowerShell supports macOS 10.12 and higher. All packages are available on our GitHub [releases][]
+PowerShell supports macOS 10.12 and higher. PowerShell 7.0.3 or higher and PowerShell Preview
+7.1.0-preview.6 or higher require macOS 1.13 and higher. All packages are available on our GitHub [releases][]
```

Comment:
In final doc, we won't want it to say "-preview.6".  Also, 1.13 should be 10.13.

Reply:
Fixed